spec.html

```diff
@@ -39143,6 +38882,30 @@ <h1>GeneratorYield ( _iterNextObj_ )</h1>
           1. NOTE: This returns to the evaluation of the operation that had most previously resumed evaluation of _genContext_.
         </emu-alg>
       </emu-clause>
+
+      <emu-clause id="sec-yield" aoid="Yield">
+        <h1>Yield ( _value_ )</h1>
+        <p>The abstract operation Yield takes argument _value_ (an ECMAScript language value). It performs the following steps when called:</p>
+        <emu-alg>
+          1. Let _generatorKind_ be ! GetGeneratorKind().
+          1. If _generatorKind_ is ~async~, then return ? AsyncGeneratorYield(_value_).
```

Comment:
```suggestion
          1. If _generatorKind_ is ~async~, return ? AsyncGeneratorYield(_value_).
```

(This is the thing the linter is complaining about.)